### PR TITLE
Feature/issue counts by status chart

### LINKED ISSUE #206: Create chart for opened and closed issues
Create a cumulative line chart to visualize both open and closed issues over time.
Based on #156.

@Nazarah will co-pilot

### PR DESCRIPTION
**Pull request by brylie**
Closes #206

# Proposed changes
- Add method to count issues for a given repository, grouping by status
- Add intermediary methods to count open and closed issues by date

@Digipalvelutehdas/dipor-team please review.

### CODE REVIEW

**Comment by jykae** [comment]
@brylie pushed the latest changes for this. Data should be now in the expected format for NVD3.

**Comment by brylie** [comment]
@jykae thanks!

**Comment by brylie** [comment]
@frenchbread, @elnzv, or @jykae, will one of you please review this PR?

**Comment by frenchbread** [comment]
I'm on it.

**Comment by bajiat** [comment]
@frenchbread Thanks for volunteering! We are waiting for this issue be merged to make the release for Sprint 5 and we would like to do that today. So are you available for reviewing the PR in the next couple of hours?

**Comment by bajiat** [comment]
PR review reassigned, because we need this change for sprint 5 release and deployment.

**Comment by jykae** [comment]
Reviewing

File: integrations/github/issues/server/methods.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 import { Github } from '/integrations/github/server/github';
+import d3 from 'd3';
+import moment from 'moment';
```
Comment (frenchbread) [inline integrations/github/issues/server/methods.js]: Why do you use `moment` from NPM when there is a package for meteor `momentjs:moment`?

**Comment by frenchbread** [comment]
For some reason I got two charts showing up for each data source which look similar:
screen shot 2016-05-03 at 12 49 30

**Comment by jykae** [comment]
For me this looks ok

**Comment by frenchbread** [comment]
@jykae Do you also get two charts?

**Comment by jykae** [comment]
@frenchbread you can select regions by dragging

**Comment by frenchbread** [comment]
@jykae Oh, I see. The second one is a range chart..

**Comment by jykae** [comment]
Colors chosen and merged.

**Pull request merged**